libs/data-service-generator/src/prepare-context.ts

```diff
@@ -63,6 +64,22 @@ export async function prepareContext(
   }
 
   const pluginsWithDefaultPlugins = prepareDefaultPlugins(resourcePlugins);
+
+  await createLog({
+    level: "info",
+    message: `The code generation process working with the plugins:`,
+  });
+  for (const plugin of pluginsWithDefaultPlugins) {
+    try {
+      await createLog({
+        level: "info",
+        message: `   ${plugin.npm}@${plugin.version}`,
+      });
+    } catch (error) {
+      logger.error(`Error to log plugin ${plugin}`);
+    }
+  }
```

Comment:
this is not the right place for the logs. I would expect to see the log in the actual place where the plugins are being used, including an option to log failure to install. 

Something like this (very high level of course) 

try
CreateLog("Installing Plugin" + plugin)
InstallPlugin(plugin)
CreateLog("Successfully Installed plugin" + plugin)
catch
CreateLog("Failed to installed plugin" + plugin + ex)
end try